### DISCUSSION TITLE
Very little space in front of meters

### DISCUSSION THREAD

**Discussion by DanielWe2** [jomjol/AI-on-the-edge-device#1025] [General] [answered]
Hi,

I would like to read two water meters with very little space in the cabinet.

There are a little over 5 cm free space in front of the meters. The numbers are a little bit recessed. It can properly not even focus on 5 cm, right?

In theory I could turn the meters. But this is only for rent I don't think my landlord would like me rotating them (And I might break the "seal")

The only other option I can think of is to build somethijg with mirrows but that seems really complicated.

Any ideas?

![20220917_132330](https://user-images.githubusercontent.com/19294888/190854540-c26a2463-4720-49c3-b504-515b7cb0a20f.jpg)

(and yes I know those are smart meters that are transmitting the data using LoRa but I think without the key that is not usable for me).

**Comment by jomjol** [comment, marked as answer]
There is a housing for such a situation with a mirror inside:
https://www.thingiverse.com/thing:5028229

**Reply by DanielWe2** [reply]
Wow, thanks didn't expect such a quick answer.

That looks a little bit complicated and seems to need some change for the form of my meters. Need to learn more about 3d modeling. But good to know there is a solution. Could you by any chance upload the source cad files of those stls?

**Comment by pos-ei-don** [comment]
I use exact this housing. It works great.

**Comment by nliaudat** [comment]
You could try to fix it on the door ....

![2022-09-17 17 27 33](https://user-images.githubusercontent.com/6782613/190864519-5bc665f6-e71c-4023-b3f7-ab88f01211df.jpg)

here is the STL : https://www.dropbox.com/s/pk2z1vzr0g8haej/ESP32-CAM_v6_hot-water.stl?dl=0

I can give you the openscad if you want.

**Reply by DanielWe2** [reply]
Thank you. I am using your model for my other meter (Screwed it to my own model ) to hold the esp. (see #1115 for a photo).

I have just one problem:  The ESP doesn't real lay flat inside the case and can wobble around. What did you do about that? Am I missing a part?

**Reply by nliaudat** [reply]
You'll find the cover part stl with openscad too here : 
[box.zip](https://github.com/jomjol/AI-on-the-edge-device/files/9732039/box.zip)

**Comment by DanielWe2** [comment]
A quick updated.  I used a remixed mirror design from thingyverse and combined it with my own model that mounts it to the meters itself.  

The mount:

![20221021_213003](https://user-images.githubusercontent.com/19294888/198876027-377e74df-01e4-492b-b0b9-ef0a7a71115b.jpg)

The final result:
![20221023_153908](https://user-images.githubusercontent.com/19294888/198876041-2bd49d69-80f1-45a3-8e8e-1c30b6a2c42a.jpg)

Thanks everyone.
